Nos/Views/Profile/ProfileKnownFollowersView.swift

```diff
@@ -5,25 +5,69 @@ struct ProfileKnownFollowersView: View {
     var knownFollowers: [Follow]
     var followers: [Follow]
 
+    private func attributedText(from stringResource: LocalizedStringResource) -> AttributedString {
+        let attributedString = AttributedString(localized: stringResource)
+        let bold = InlinePresentationIntent.stronglyEmphasized.rawValue
+        return attributedString.replacingAttributes(
+                AttributeContainer(
+                    [.inlinePresentationIntent: bold]
+                ),
+                with: AttributeContainer(
+                    [.foregroundColor: UIColor(Color.primaryTxt)]
+                )
+            )
+    }
+
     var body: some View {
         HStack {
             if let second = knownFollowers[safe: 1]?.source {
-                StackedAvatarsView(avatarUrls: [first.profilePhotoURL, second.profilePhotoURL])
+                StackedAvatarsView(
+                    avatarUrls: [first.profilePhotoURL, second.profilePhotoURL]
+                )
```

Comment:
I know our SwiftLint line length limit is 120 and it doesn't look like anything made this line longer. Do we need to discuss changing our line length limit? I'm fine with breaking lines apart for readability or, obviously, when we go above the limit, but here it doesn't look like either of those applies.

Reply:
I prefer to write lines shorter than 80. I don't want to enforce that as a rule, it's just a personal preference when writing code. I know we don't have previews anymore, but when we had it, that section, plus the file navigator on the left, plus the editor on the center, and without a external screen... code looks nicer if the limit is 80, that's why I prefer that in general. But I agree the limit should be around 120.